src/app/tests/TestOperationalDeviceProxy.cpp

```diff
@@ -1,118 +0,0 @@
-#include <system/SystemLayerImpl.h>
-#include <transport/SessionManager.h>
-#include <transport/TransportMgr.h>
-#include <transport/raw/PeerAddress.h>
-#include <transport/raw/UDP.h>
-
-using namespace chip;
-using namespace chip::Transport;
-using namespace chip::Messaging;
-using namespace chip::Credentials;
-
-#if INET_CONFIG_ENABLE_IPV4
-namespace {
-
-using TestTransportMgr = TransportMgr<Transport::UDP>;
-
-void TestOperationalDeviceProxy_EstablishSessionDirectly(nlTestSuite * inSuite, void * inContext)
-{
-    // TODO: This test appears not to be workable since it does not init the fabric table!!!
-    Platform::MemoryInit();
-    TestTransportMgr transportMgr;
-    SessionManager sessionManager;
-    SimpleSessionResumptionStorage sessionResumptionStorage;
-    ExchangeManager exchangeMgr;
-    Inet::UDPEndPointManagerImpl udpEndPointManager;
-    System::LayerImpl systemLayer;
-    // Heap-allocate the fairly large FabricTable so we don't end up with a huge
-    // stack.
-    FabricTable * fabrics     = Platform::New<FabricTable>();
-    const FabricInfo * fabric = fabrics->FindFabricWithIndex(1);
-    VerifyOrDie(fabric != nullptr);
-    secure_channel::MessageCounterManager messageCounterManager;
-    chip::TestPersistentStorageDelegate deviceStorage;
-    GroupDataProviderImpl groupDataProvider;
-
-    systemLayer.Init();
-    udpEndPointManager.Init(systemLayer);
-    transportMgr.Init(UdpListenParameters(udpEndPointManager).SetAddressType(Inet::IPAddressType::kIPv4).SetListenPort(CHIP_PORT));
-    sessionManager.Init(&systemLayer, &transportMgr, &messageCounterManager, &deviceStorage);
-    sessionResumptionStorage.Init(&deviceStorage);
-    exchangeMgr.Init(&sessionManager);
-    messageCounterManager.Init(&exchangeMgr);
-    groupDataProvider.SetPersistentStorage(&deviceStorage);
-    VerifyOrDie(groupDataProvider.Init() == CHIP_NO_ERROR);
-    // TODO: Set IPK in groupDataProvider
-
-    CASEClientInitParams params = {
-        .sessionManager           = &sessionManager,
-        .sessionResumptionStorage = &sessionResumptionStorage,
-        .exchangeMgr              = &exchangeMgr,
-        .fabricInfo               = fabric,
-        .groupDataProvider        = &groupDataProvider,
-    };
-    NodeId mockNodeId = 1;
-    OperationalDeviceProxy device(params, PeerId().SetNodeId(mockNodeId));
-    Inet::IPAddress mockAddr;
-    Inet::IPAddress::FromString("127.0.0.1", mockAddr);
-    PeerAddress addr = PeerAddress::UDP(mockAddr, CHIP_PORT);
-    device.UpdateAddress(addr);
-
-    NL_TEST_ASSERT(inSuite, device.Connect(nullptr, nullptr) == CHIP_NO_ERROR);
-
-    device.Clear();
-    messageCounterManager.Shutdown();
-    exchangeMgr.Shutdown();
-    sessionManager.Shutdown();
-    Platform::Delete(fabrics);
-    transportMgr.Close();
-    udpEndPointManager.Shutdown();
-    systemLayer.Shutdown();
-    Platform::MemoryShutdown();
-}
-
-// clang-format off
-const nlTest sTests[] =
```

Comment:
so there is really no value in these test cases?

Reply:
It has not been run since at least Jul 2022 (when it would not have compiled anymore for sure due to header changes) ... it also seems to just try to establish some connection.

Overall it does not seem like a real "unit test" and establishing connections are very tested as E2E tests. If this has not been run for 1.5+ years it seemed way more of a pain to try to resurect it than to just officially not run it anymore.